### PR TITLE
[Merged by Bors] - 121 add documentation hbase hdfs demo

### PR DESCRIPTION
**Pull request by snocke**
## Description

This adds documentation to the hdfs-hbase demo.

## Review Checklist
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Review by sbernauer — COMMENTED** [review]
I did some changes to the wording and pushed it to `121-add-documentation-hbase-hdfs-demo-review` feel free to merge that into your branch if you agree with the changes.

It would we great id you could also show the HBase webinterface and the created table there.

Also showing the files in hdfs might help users follow the demo even better. You could use the hdfs command or the Namenode webinterface for this.

File: docs/modules/ROOT/pages/demos/hbase-hdfs-load-cycling-data.adoc [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+[LocalJobRunner Map Task Executor #0] mapred.Task (Task.java:commit(1421)) - Task attempt_local60745921_0001_m_000000_0 is allowed to commit now
+[LocalJobRunner Map Task Executor #0] output.FileOutputCommitter (FileOutputCommitter.java:commitTask(609)) - Saved output of task 'attempt_local60745921_0001_m_000000_0' to file:/tmp/hadoop/mapred/staging/stackable339030898/.staging/_distcp-1760904616/_logs
+[LocalJobRunner Map Task Executor #0] mapred.LocalJobRunner (LocalJobRunner.java:statusUpdate(634)) - 100.0% Copying s3a://public-backup-nyc-tlc/cycling-tripdata/demo-cycling-tripdata.csv.gz to hdfs://hdfs/data/raw/demo-cycling-tripdata.csv.gz
+----
+
+== The second Job
+The second Job consists of 2 steps.
+First, we use `org.apache.hadoop.hbase.mapreduce.ImportTsv` (see https://hbase.apache.org/book.html#importtsv) to create a table and Hfiles.
+Hfile is an Hbase dedicated file format which is performance optimized for hbase. It stores meta information about the data and thus increases the performance of hbase
+When connecting to the hbase master and opening a `bin/hbase shell` and executing `list`, you will see the created table. However, it'll contain 0 rows at this point.
```
Comment (sbernauer) [inline docs/modules/ROOT/pages/demos/hbase-hdfs-load-cycling-data.adoc:73]: Could you please provide the command for this?
Something like `kubectl exec -it hbase-master-0-or-so -- bin/hbase shell`

File: docs/modules/ROOT/pages/demos/hbase-hdfs-load-cycling-data.adoc [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+[LocalJobRunner Map Task Executor #0] mapred.LocalJobRunner (LocalJobRunner.java:statusUpdate(634)) - 100.0% Copying s3a://public-backup-nyc-tlc/cycling-tripdata/demo-cycling-tripdata.csv.gz to hdfs://hdfs/data/raw/demo-cycling-tripdata.csv.gz
+----
+
+== The second Job
+The second Job consists of 2 steps.
+
+First, we use `org.apache.hadoop.hbase.mapreduce.ImportTsv` (see https://hbase.apache.org/book.html#importtsv[ImportTsv Docs]) to create a table and Hfiles.
+Hfile is an Hbase dedicated file format which is performance optimized for hbase. It stores meta information about the data and thus increases the performance of hbase
+When connecting to the hbase master and opening a `bin/hbase shell` and executing `list`, you will see the created table. However, it'll contain 0 rows at this point.
+You can connect to the shell via `kubectl exec -it hbase-master-default-0 -- bin/hbase shell`.
```
Comment (sbernauer) [inline docs/modules/ROOT/pages/demos/hbase-hdfs-load-cycling-data.adoc:87]: Please put that in a [source,console] block. The rendered docs screwed up the command somehow

File: docs/modules/ROOT/pages/demos/hbase-hdfs-load-cycling-data.adoc [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+{NAME => 'start_station_id', BLOOMFILTER => 'ROW', IN_MEMORY => 'false', VERSIONS => '1', KEEP_DELETED_CELLS => 'FALSE', DATA_BLOCK_ENCODING => 'NONE', COMPRESSION => 'NONE', TTL => 'FOREVER', MIN_VERSIONS => '0', BLOCKCACHE => 'true', BLOCKSIZE => '65536', REPLICATION_SCOPE => '0'}
+{NAME => 'start_station_name', BLOOMFILTER => 'ROW', IN_MEMORY => 'false', VERSIONS => '1', KEEP_DELETED_CELLS => 'FALSE', DATA_BLOCK_ENCODING => 'NONE', COMPRESSION => 'NONE', TTL => 'FOREVER', MIN_VERSIONS => '0', BLOCKCACHE => 'true', BLOCKSIZE => '65536', REPLICATION_SCOPE => '0'}
+{NAME => 'started_at', BLOOMFILTER => 'ROW', IN_MEMORY => 'false', VERSIONS => '1', KEEP_DELETED_CELLS => 'FALSE', DATA_BLOCK_ENCODING => 'NONE', COMPRESSION => 'NONE', TTL => 'FOREVER', MIN_VERSIONS => '0', BLOCKCACHE => 'true', BLOCKSIZE => '65536', REPLICATION_SCOPE => '0'}
+----
+
+== The Hbase UI
+The Hbase web UI will give you information on status and metrics of your Hbase cluster. See below for the startpage.
```
Comment (sbernauer) [inline docs/modules/ROOT/pages/demos/hbase-hdfs-load-cycling-data.adoc:137]: Can you please tell the user how to reach the UI? Something like `k port-forward hbase-master-default-0 16010`

File: docs/modules/ROOT/pages/demos/hbase-hdfs-load-cycling-data.adoc [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+image::demo-hbase-hdfs-load-cycling-data/hbase-ui-start-page.png[]
+
+From the startpage you can check more details. For example details on the created table.
+
+image::demo-hbase-hdfs-load-cycling-data/hbase-table-ui.png[]
+
+== The HDFS UI
+You can also see HDFS details via a UI. Below you will see the overview of your HDFS cluster
```
Comment (sbernauer) [inline docs/modules/ROOT/pages/demos/hbase-hdfs-load-cycling-data.adoc:146]: Can you please tell the user how to reach the UI? Currently we need port-forward as the hdfs nigthly operator fixed the service labels. Next release we can use the services list command output

**Review by sbernauer — COMMENTED** [review]
LGTM overall

**Comment by snocke** [comment]
bors merge

**Pull request closed without merging**